demo/src/main/AndroidManifest.xml

```diff
@@ -27,8 +29,18 @@
                 <category android:name="android.intent.category.DEFAULT" />
                 <category android:name="android.intent.category.BROWSABLE" />
             </intent-filter>
+            <intent-filter android:autoVerify="true">
+                <action android:name="android.intent.action.VIEW" />
+
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
+
+                <data android:scheme="https" />
+                <data android:host="mobile-sdk-demo-site-838cead5d3ab.herokuapp.com" />
+            </intent-filter>
         </activity>
-        <activity android:name=".DemoActivitySingleTop"
+        <activity
+            android:name=".DemoActivitySingleTop"
```

Comment:
I'll need to figure out how to handle the DemoActivitySingleTop activity with the same defined app link. I'll follow up on this in a follow-up PR.